participating.md

```diff
@@ -1,5 +1,70 @@
+
+The project is entirely open source and composability is a core design principle - we want you to be able to create your own models and plug them in to our framework, or pick from a broad universe of open source models created by others.
+
+For **Carbon Hack** we are challenging participants to build solutions that align to and extend the capability of IF. 
+
+## 💚 I've got an idea, is it any good?
+
+- **Idea criteria**: To have a good idea, you should consider how your idea aligns with the following criteria:
+    - **Relevance**: How relevant is your idea to the theme of Carbon Hack 24 and the mission of the Green Software Foundation?
+    - **Innovation**: How innovative is your idea in terms of using IF to measure and reduce the negative environmental impacts of software?
+    - **Feasibility**: How feasible is your idea in terms of technical, financial, and time resources?
+    - **Scalability**: How scalable is your idea in terms of potential impact and applicability to different contexts, industries and domains?
+    - **Replicability**: How replicable is your idea in terms of sharing best practices and learnings with other software practitioners and stakeholders?
+- **Idea validation**: To validate your idea, you should use the IF tools and resources to estimate the environmental impact of your software solution and compare it with the baseline or alternative solutions. You should also seek feedback from your peers, mentors, and experts on the Carbon Hack 24 platform and community.
+
+
+## 💚 What prize category should we go for?
+#### **Prize categories**: _There are four prize categories for Carbon Hack 24:_
+- 💧 Beyond Carbon
+Any submission that demonstrates the application of Impact Framework beyond carbon. (e.g. water, waste, air quality).
+
+- 📦 Best Model
+A model plugin that best supports and enhances the Impact Framework model plugin ecosystem.
+
+- 📝 Best Content
+Best piece of content published about Impact Framework, an article, tutorial, video or how to guide.
+
+- ✨ Best Contribution to the Framework
+Best contribution to the Impact Framework tool itself. Adding a new feature or enhancing an existing one.
+
+#### We are also keen to broaden the opportunities for those who are studying:
+- 🎓 Best Undergraduate
+Specific prize for Undergraduate Students, drawn from across the main categories.
+
+
+#### **Prize selection**: To select the prize category that best suits your idea, you should consider the following questions:
+- **What is the main goal of your idea?**: 
+- **What is the main strength of your idea?**:
+- **What is the main challenge of your idea?**: Is it the relevance, the feasibility, the scalability, or the replicability of your idea?
+- **What is the main audience of your idea?**: Is it software practitioners, software users, software stakeholders, or the general public?
+ 
+
+## 💚 I need help on my project
```

Comment:
Be more concrete.

- Ask an initial questions in your project issue and the hackathon team will make sure to highlight them to the IF team to help answer.
- If you have any broader questions or want to get feedback from the community ask in the ask anything channel in the discussions forum, we'd always rather you asked questions publically as that helps the most people.
- After submitting a project you and all of your team will be added to a mail thread with the hackathon leaders, if you have any quetions you can ask there aswell.